File: csharp/main/Implementation/PropertyNode.cs
using System;
using System.Collections.Generic;
using JetBrains.Annotations;

namespace Deltix.Luminary.Implementation
{
	internal class PropertyNode : ItemNode
	{
		public PropertyNode([NotNull] String name, [NotNull] String type, [CanBeNull] List<String> comments)
			: base(name, comments)
		{
			Type = type;
		}

		public String Type { get; }
	}
}

File: java/src/main/java/deltix/luminary/implementation/PropertyNode.java
package deltix.luminary.implementation;

import java.util.List;

public class PropertyNode extends ItemNode {
    private final String type;

    protected PropertyNode(String name, String type, List<String> comments) {
        super(name, comments);
        this.type = type;
    }

    public String getType() {
        return type;
    }
}
